modules/calypsoify/mods-gutenberg.js

```diff
@@ -25,4 +39,26 @@ jQuery( function( $ ) {
 		var href = $( this ).attr( 'href' );
 		$( this ).attr( 'href', href.replace( '&classic-editor', '' ) );
 	} );
+
+	if ( inIframe() ) {
```

Comment:
We already removed this code form WP.com (D27601-code) and added it to `@automattic/wpcom-block-editor` (https://github.com/Automattic/wp-calypso/pull/32667). 

This means that JP is already loading a script with this same logic, so I don't think changes on this file are really needed.

Reply:
That makes sense, updated!